core/src/main/kotlin/behavior/GlobalApplicationCommandBehavior.kt

```diff
@@ -37,6 +38,19 @@ interface ApplicationCommandBehavior : Entity {
      * @throws [RestRequestException] when something goes wrong during the request.
      */
     suspend fun delete()
+
+    /**
+     * Updates the permissions for this command on the guild corresponding to [guildId].
+     *
+     * @throws [RestRequestException] when something goes wrong during the request.
+     */
+    suspend fun editPermissions(
```

Comment:
Move this to `GuildApplicationCommandBehavior which will always have the guildId